examples/point-cloud-ply/app.js

```diff
@@ -14,7 +14,6 @@ class Example extends PureComponent {
     super(props);
 
     this._onResize = this._onResize.bind(this);
-    this._onInitialized = this._onInitialized.bind(this);
```

Comment:
bug: _onInitialized is no longer used after switching to the new orbit-controller